### ISSUE TITLE
ML: Deep Model #1 (24 Layers): Configuration#2: Training model and store the results in the directory for 30 and 50 epochs

### ISSUE THREAD

**Issue by hmehryar** [hmehryar/Hm.JetscapeMl#266]
Model Architect: 

- 4Conv(Conv+Conv+MaxPool+Dropout)
- 4Dense(Dense+Dropout)

1. Put the training on the Grid by using jetscape-ml-tensorflow-cnn.sh
2. save the model and the evaluation data in the power point [slides](https://docs.google.com/presentation/d/1H7D-TKbmZjwNFNcf8Ecuz3HLl1veCZVL/edit?usp=sharing&ouid=108370064464811889896&rtpof=true&sd=true)
3. update the diagrams data on the power point slides

**Comment by hmehryar** [comment]
epoch 30

Epoch 30/30
839936/840000 [============================>.] - ETA: 0s - loss: 1.0687 - accuracy: 0.9124  
Epoch 00030: val_accuracy did not improve from 0.91058
840000/840000 [==============================] - 309s 368us/sample - loss: 1.0687 - accuracy: 0.9124 - val_loss: 1.1201 - val_accuracy: 0.8626

**Comment by hmehryar** [comment]
50 epochs

Epoch 43/50
839936/840000 [============================>.] - ETA: 0s - loss: 0.6015 - accuracy: 0.9128  
Epoch 00043: val_accuracy did not improve from 0.92174
840000/840000 [==============================] - 318s 378us/sample - loss: 0.6015 - accuracy: 0.9128 - val_loss: 0.6371 - val_accuracy: 0.8845
Epoch 00043: early stopping

**Comment by hmehryar** [comment]
port to excel for Overleaf
